component/avatar/avatar.css

```diff
@@ -1,10 +1,3 @@
-@import url('/style.css');
- @import url('/component/slidebaar.css');
- @import url('/component/alert/alert.css');
-*{
-    margin: 0px;
-    padding: 0px;
-}
 .Extra-large-img img{
```

Comment:
class naming should start with small letter.

Reply:
I have editted class name